Modules/AppointmentSlots/Resources/views/select_appointments/interview_schedule_error_message.blade.php

```diff
@@ -0,0 +1,19 @@
+@extends('appointmentslots::layouts.master')
+
+@section('content')
+
+<div>
+    <div class="placing-of-content text-center">
+        <svg class="exclamation-icon h-44 w-44">{!! file_get_contents(public_path('icons/exclamation-circle.svg')) !!}</svg>

```

Comment:
You should not use svg inside a SVG tag as the file already has SVG tag in it.
```suggestion
        <div class="mx-auto exclamation-icon h-44 w-44">{!! file_get_contents(public_path('icons/exclamation-circle.svg')) !!}</div>

```